todo.org

```diff
@@ -68,3 +68,17 @@ CLOSED: [2020-06-24]
 - [X] Implement =snapcraft list-tracks=
 - [ ] Add documentation for track management to the [[https://snapcraft.io/docs/release-management][Release management]] docs.
 
+*** TODO [[file:specifications/configurable-apt-mirror.org][Configurable APT Mirror]] [0/7]
+- [ ] Add =--snapcraft-apt-mirror= command-line argument with matching
+  =SNAPCRAFT_APT_MIRROR= environment variable.
+- [ ] Replace usage of =SNAPCRAFT_BUILD_EVNIRONMENT_PRIMARY_MIRROR= with
+  =SNAPCRAFT_APT_MIRROR=.
+- [ ] Remove configurable security mirror.
```

Comment:
The is no mention that this is configurable in the text

Reply:
There's a broken attempt to make it configurable to remove (it uses SNAPCRAFT_BUILD_ENVIRONMENT_PRIMARY_MIRROR).